tests/integration/messages/create_messages/test_field_validation.py

```diff
@@ -312,28 +273,187 @@ def test_null_personalisation(bearer_token_int, correlation_id, personalisation)
     """
     .. include:: ../../partials/validation/test_invalid_personalisation.rst
     """
-    resp = requests.post(f"{INT_URL}{MESSAGES_ENDPOINT}", headers={
-        "Authorization": bearer_token_int.value,
-        **headers,
-        "X-Correlation-Id": correlation_id
-    }, json={
-        "data": {
-            "type": "Message",
-            "attributes": {
-                "routingPlanId": "b838b13c-f98c-4def-93f0-515d4e4f4ee1",
-                "messageReference": "invalid",
-                "recipient": {
-                    "nhsNumber": "9990548609",
-                    "dateOfBirth": "2023-01-01"
-                },
-                "personalisation": personalisation
-            }
-        }
-    })
+    data = Generators.generate_valid_create_message_body("int")
+    data["data"]["attributes"]["personalisation"] = personalisation
+    resp = requests.post(
+        f"{INT_URL}{MESSAGES_ENDPOINT}",
+        headers={
+            "Authorization": bearer_token_int.value,
+            **headers,
+            "X-Correlation-Id": correlation_id
+        },
+        json=data
+    )
 
     Assertions.assert_error_with_optional_correlation_id(
         resp,
         400,
         Generators.generate_null_value_error("/data/attributes/personalisation"),
         correlation_id
     )
+
+
+@pytest.mark.inttest
+@pytest.mark.parametrize("correlation_id", CORRELATION_IDS)
+def test_invalid_sms_contact_details(bearer_token_int, correlation_id):
+    """
+    .. include:: ../../partials/validation/test_invalid_contact_details_sms.rst
+    """
+    data = Generators.generate_valid_create_message_body("int")
+    data["data"]["attributes"]["recipient"]["contactDetails"] = {"sms": "07700900002"}
+    resp = requests.post(
+        f"{INT_URL}{MESSAGES_ENDPOINT}",
+        headers={
+            "Authorization": bearer_token_int.value,
+            **headers,
+            "X-Correlation-Id": correlation_id
+        },
+        json=data
+    )
+
+    Assertions.assert_error_with_optional_correlation_id(
+        resp,
+        400,
+        Generators.generate_invalid_value_error_custom_detail(
+            "/data/attributes/recipient/contactDetails/sms",
```

Comment:
Are these tests likely to break with the introduction of this ticket? https://nhsd-jira.digital.nhs.uk/browse/CCM-6415